src/components/shared/forms/tooltip.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import { joinClasses } from 'utilities';
+
+import './styles/tooltip.css';
+
+export class Tooltip extends Component {
+  render() {
+    const { target, children, position } = this.props;
+
+    return (
+      <div class="tooltip-container">
+        <div class={joinClasses('tooltip', position)}>
+          <div class="tooltip-content">
+            {children}
+          </div>
+          <div class="tooltip-arrow"></div>
```

Comment:
nit: since this is jsx and not html your can do this `<div class="tooltip-arrow" />`